gradle.properties

```diff
@@ -3,7 +3,7 @@ org.gradle.parallel=false
 org.gradle.jvmargs=-Xms512m -Xmx512m
 scalaVersion=2.13.13
 kafkaVersion=3.5.1
-zookeeperVersion=3.8.3
-nettyVersion=4.1.100.Final
-jettyVersion=9.4.53.v20231009
+zookeeperVersion=3.8.4
+nettyVersion=4.1.108.Final
+jettyVersion=9.4.54.v20240208
```

Comment:
There's a newer version for jetty, 9.4.54 has a medium one: https://nvd.nist.gov/vuln/detail/CVE-2024-8184
9.4.56.v20240826

Reply:
I think moving to `9.4.56.v20240826` should be safe